inc/class-mltools-elementor-config-generator.php

```diff
@@ -0,0 +1,269 @@
+<?php
+
+class MLTools_Elementor_Config_Generator {
+
+	public function __construct() {
+		if ( class_exists( 'Sitepress' ) && is_plugin_active( 'elementor/elementor.php' ) ) {
+			add_action( 'add_meta_boxes', [ $this, 'register_meta_box' ] );
+		}        
+	}
+	
+	public function register_meta_box() {
+		$screens = get_post_types();
+		global $post;
+		$elementor_data = get_post_meta( $post->ID, '_elementor_data', true );
+		
+		if ( ! empty( $elementor_data ) ) {
+			foreach ( $screens as $screen ) {
+				add_meta_box(
+					'wpmlpb_config_generator_box', 
+					'WPML - Config Generator for Elementor', 
+					[ $this, 'meta_box_html' ], 
+					$screen
+				);
+			}
+		}
+	}
+	
+
+	public function get_widgets_blacklist() {
+		$widgets_blacklist = [];
+		
+		if ( class_exists( 'WPML_Elementor_Translatable_Nodes' ) ) {
+			$wpml_elementor_translatable_nodes = new WPML_Elementor_Translatable_Nodes();
+			$default_widgets                   = $wpml_elementor_translatable_nodes->get_nodes_to_translate();
+			$default_widgets                   = apply_filters( 'wpml_elementor_widgets_to_translate', $default_widgets );
+		
+			foreach ( $default_widgets as $key => $value ) {
+				$widgets_blacklist[] = $key;
+			}
+		}		
+		
+		return $widgets_blacklist;
+	}
+	
+	public function get_settings_blacklist() {
+		$blacklist = [];
+		
+		$blacklist[] = 'is_external';
+		$blacklist[] = 'nofollow';
+		$blacklist[] = 'custom_attributes';
+		
+		return $blacklist;
+	}
+	
+	public function get_widgets_list( $elements ) {
+		$widgets = [];
+		
+		foreach ( $elements as $element ) {
+			if ( $element->elType === 'widget' ) {
+				$widgetType                    = $element->widgetType;
+				$settings                      = (array) get_object_vars( $element->settings );
+		
+				foreach ( $settings as $field_key => $field_value ) {
+					$settings[ $field_key ]   = $this->get_field_from_widget( $field_key, $field_value, $widgetType );
```

Comment:
This one has too many spaces @diiegopereira 
```suggestion
					$settings[ $field_key ] = $this->get_field_from_widget( $field_key, $field_value, $widgetType );
```